src/main/java/frc/robot/subsystems/Acquisition.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkFlex;
+import com.revrobotics.spark.SparkLimitSwitch;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkFlexConfig;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.AcquisitionConstants;
+import frc.robot.constants.NeoMotorConstants;
+
+public class Acquisition extends SubsystemBase {
+    private static DigitalInput topLaser;
+    private static DigitalInput bottomLaser;
+    private static SparkFlex acquisitionMotor;
+    private static SparkLimitSwitch algaeSwitch;
```

Comment:
These should not be static and should have their constructor called here, not in the class constructor